services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/client_api.py

```diff
@@ -113,10 +157,8 @@ async def setup_api_client(app: FastAPI) -> None:
     app.state.dynamic_sidecar_api_client = DynamicSidecarClient(app)
 
 
-async def shutdown_api_client(app: FastAPI) -> None:
+async def shutdown_api_client(app: FastAPI) -> None:  # pylint: disable=unused-argument
     logger.debug("dynamic-sidecar api client shutdown")
-    dynamic_sidecar_api_client = app.state.dynamic_sidecar_api_client
-    await dynamic_sidecar_api_client.close()
```

Comment:
why you need this method if it does nothing?

Reply:
indeed, removed